Code/DistGeom/DistGeomUtils.h

```diff
@@ -148,6 +148,22 @@ RDKIT_DISTGEOMETRY_EXPORT ForceFields::ForceField *construct3DForceField(
 
   <b>NOTE:</b> the caller is responsible for deleting this force field.
 
+*/
+RDKIT_DISTGEOMETRY_EXPORT ForceFields::ForceField *construct3DForceField(
+    const BoundsMatrix &mmat, RDGeom::Point3DPtrVect &positions,
+    const ForceFields::CrystalFF::CrystalFFDetails &etkdgDetails,
+  const std::map<std::pair<unsigned int, unsigned int>, double> &TO_BE_NAMED_Ptr);
```

Comment:
I'm pretty sure you just forgot to rename this parameter

Reply:
yep :P

Reply:
and I added regression tests for torsions preferences now. I already had regression test for CPCIs before.